nemo/collections/nlp/data/datasets/datasets_utils/data_preprocessing.py

```diff
@@ -143,8 +143,8 @@ def read_csv(file_path):
 
 
 def get_dataset(files, dev_split=0.1):
-    entity2value, value2entity = get_entities(files)
-    data, slots, intents = get_data(files, entity2value, value2entity)
+    # entity2value, value2entity = get_entities(files)
```

Comment:
can we delete this line?

Reply:
I keep it there because Purnendu wrote that code and I am not sure why that line is there. Possibly for debugging? I am going to ask Purnendu to clean up the code. @purn3ndu